docs/index.rst

```diff
@@ -1,22 +1,25 @@
-.. TensorNetwork documentation master file, created by
-   sphinx-quickstart on Thu Jun  6 14:53:19 2019.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
-Welcome to TensorNetwork's documentation!
+TensorNetwork reference documentation
 =========================================
 
 .. toctree::
-   :maxdepth: 4
-   :caption: Contents:
+   :maxdepth: 1
+   :caption: Tutorials
 
    tutorial
+   basic_api_tutorial
+   Tensor Networks inside Neural Networks using Keras <https://colab.research.google.com/drive/1JUh84N5sbfQYk6HWowWCGl0IZ1idQi6z>
```

Comment:
Can you actually include the ipython notebook like you did with the basic API tutorial?

Reply:
Done as well